type_description_interfaces/msg/IndividualTypeDescription.msg

```diff
@@ -0,0 +1,6 @@
+# Represents a single type, without the types it references, if any.
+
+# Name of the type.
+string type_name
+# Fields of the type.
+Field[] fields
```

Comment:
Consider adding constants/enums/annotations here, being careful to consider impact to hashing.

Reply:
I added a todo in the README for this.